testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -85,7 +85,7 @@
     "XTC_sub_sol",
     "XYZ", "XYZ_psf", "XYZ_bz2",
     "XYZ_mini", "XYZ_five", # 3 and 5 atoms xyzs for an easy topology
-    "TXYZ", "ARC", "ARC_PBC",        # Tinker files
```

Comment:
I am not sure why ARC_PBC was removed. This might have been a merge accident because it is present in current develop. *Please add it back!* The line should be
```python
 "TXYZ", "ARC", "ARC_PBC",        # Tinker files
```

Reply:
Make sure that you pull from your branch – maybe you didn't get the merge commit?